### ISSUE TITLE
Touch scrolling on mobile devices

### ISSUE THREAD

**Issue by elspeth-rabid** [openseadragon/openseadragon#1528]
Hi!

Thanks for the amazing library!

I am having one tricky UX problem on _mobile_ devices. I have a openseadragon component at the top of a page, full width, and portrait images can take up most or all of the mobile screen space. The user needs to be able to scroll past the openseadragon component, to the information about the image.

Touch scrolling zooms the component by default, so I successfully disabled that with http://openseadragon.github.io/docs/OpenSeadragon.html#.GestureSettings.
Now when there is a touch drag/scroll event on an image that is not zoomed in, nothing happens. If I zoom in on the image, touch scrolling pans the image.

If a user pans to the bottom of the image and keeps scrolling, could it start scrolling the page instead of hitting the boundary and stopping?

I have seen similar questions in other issues but none of the advice has resolved my issue.

I think this would be a great user experience! 

Any advise on how to achieve will be greatly appreciated.

**Comment by iangilman** [comment]
Yes, managing scroll and pan on mobile is tricky! 

Having the same gesture continue into a scroll sounds interesting, but I think it could be challenging. Basically you'll need to detect when the touch leaves the viewer, and start applying the movement to the window's scrollTop. This will probably require some surgery in MouseTracker. You'll also need to think about what to do if the user has multiple fingers on the screen (as part of a pinch, for instance)... Probably shouldn't translate to scroll in that case.

**Comment by rastapopougros** [comment]
Hello, it seems the Leaflet community resolved this problem in this plugin for ex : https://github.com/elmarquis/Leaflet.GestureHandling

Despite its name, this plugin manages both desktop mouse wheel and touch : 
- on desktop, it replaces wheel to zoom by Ctrl+wheel to zoom (with a nice enormous message in many languages) => then the normal wheel continues to scroll the HTML !
- on mobile it replaces one finger to pan image/map by two finger (again with a nice message in many languages) => then the one finger touch continues to sroll the HTML !

I think it would be the perfect solution to imitate those behaviors, whether in a plugin or core, but with an option to "don't interfere with the scroll" (like `handleGesture: true` or else), and obviously with the same unique option solutioning  both desktop and touch.

Nobody made a plugin/option like that since all those years ?

**Comment by iangilman** [comment]
Agreed! And no, I'm not aware of a plugin for this yet. Here's some recent discussion: https://github.com/openseadragon/openseadragon/issues/2597

Would you be interested in making this feature?

**Comment by rastapopougros** [comment]
Thanks, for pointing the more recent issue (maybe this one could be close as duplicate, as long as there is a link the the one which is still open).

I'm just an end-user, and don't think I'm enough good at JS to develop the whole feature, unfortunately :)

**Comment by iangilman** [comment]
Cool. I guess just follow along with these two issues to see what developments there are.

I think I'll leave both of them open for now since they are talking about slightly different approaches.
